pkg/sql/sem/builtins/builtins.go

```diff
@@ -6823,14 +6824,71 @@ run from. One of 'mvccGC', 'merge', 'split', 'replicate', 'replicaGC',
 				if !foundRepl {
 					return nil, errors.Errorf("replica with range id %s not found on this node", rangeID)
 				}
+
 				return tree.DBoolTrue, nil
```

Comment:
What do you think about returning two values for the builtin? A success/failure boolean and string trace. That seems preferable to adding an overload. It would also let you ignore the (unstable) trace in the testlogic files.

Reply:
makes sense, I'll change this

Reply:
spoke offline, builtins only support single return values today so will have to keep the overload.